style.css

```diff
@@ -168,8 +124,8 @@ header > .logo {
   background-size: 100%;
   background-repeat: no-repeat;
   border-radius: 0 0 0 90px;
-  padding: 100px 20px 0 20;
   background-color: white;
+  filter: blur(5px);
```

Comment:
- [ ] Please remove the filter as it makes the header not readable.

![Screenshot from 2022-07-26 19-35-10](https://user-images.githubusercontent.com/33752735/181073478-2a47c763-675a-4339-81af-59418d415227.png)